src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryText.cs

```diff
@@ -353,6 +353,11 @@ internal static bool TryExpandDateTimeExcelFormatSpecifiersToStringValue(IRConte
                     var formatStr = ExpandDateTimeFormatSpecifiers(format, culture);
                     result = new StringValue(irContext, dateTime.ToString(formatStr, culture));
                     break;
+                case "'utc'":
+                case "utc":
+                    var formatUtcStr = ExpandDateTimeFormatSpecifiers(format, culture);
+                    result = new StringValue(irContext, dateTime.ToUniversalTime().ToString(formatUtcStr, culture));
```

Comment:
>dateTime.ToUniversalTime( [](http://example.com/codeflow?start=56&length=25)

We should probably use our TimeZone service rather than rely on machine time. 

[https://github.com/microsoft/Power-Fx/blob/0992c4e31a4708c0f739d5e66f98a907d18c18b6/src/libraries/Microsoft.PowerFx.Interpreter/Environment/SymbolExtensions.cs#L18](https://github.com/microsoft/Power-Fx/blob/0992c4e31a4708c0f739d5e66f98a907d18c18b6/src/libraries/Microsoft.PowerFx.Interpreter/Environment/SymbolExtensions.cs#L18)


We might need a C# test to leverage SetTimeZone.

Reply:
Yes, i updated logic and test.